pkg/operation/botanist/component/gardeneraccess/access.go

```diff
@@ -59,6 +58,8 @@ type gardener struct {
 
 // Values contains configurations for the component.
 type Values struct {
+	// ManagedResourceName is the name of the managed resource for gardeneraccess
+	ManagedResourceName string
```

Comment:
In the GRM component, we have a `NamePrefix` field and a `GetManagedResourceName()` interface function. Why are you following another approach here?